.github/workflows/build.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         runs-on: ubuntu-latest
         strategy:
             matrix:
-                node: [8, 10, 11, 12, 13]
+                node: [11, 12, 13, 14]
```

Comment:
Should we mention in the `CHANGELOG.md` (and eventually the release notes) that we're dropping testing for Node versions 8 and 10, and are also now testing for support for v14

Reply:
yes, added in changelog